cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
this file is not used, could we delete it?

Reply:
Yes, I deleted it.